twirssi.pl

```diff
@@ -1925,7 +1926,7 @@ sub get_updates_child {
                 unless (&get_timeline( $fh, $frusers[ $fix_replies_index{$username} ],
                                                $username, $twits{$username}, \%context_cache, $is_regular )) {
                     $error++;
-                    push @error_types, 'replies';
+                    push @error_types, 'Failed to get_timeline(frusers)';
```

Comment:
My original plan for `@error_types` was to be a succinct list of _areas_ of errors, not English error messages, as evidenced by the last usage of the var (see line with `For recurring DMs errors`)